test/gsb-test-srv/proto/safebrowsing.proto

```diff
@@ -0,0 +1,467 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Correct, it's from https://github.com/google/safebrowsing/tree/master/internal/safebrowsing_proto 

I can add a link in the top of the file

I don't _believe_ I can access these types through the upstream lib but I can verify that assumption.